### ISSUE TITLE
how to use

### ISSUE THREAD

**Issue by BookmarkletsMan** [geekdictionarypro/Swagbucks-Web-Search-Bot#1]
help pls tutorial

**Comment by pearlheartgtr** [comment]
Yeah, I've tried running it and couldn't figure it out. I've been using commandline Bing Rewards bots for years with no issue. But this has no instructions at all.
